docs/ml/creating-df-kib.asciidoc

```diff
@@ -0,0 +1,32 @@
+[[creating-df-kib]]
+== Creating {dataframe-transforms}
+
+beta[]
+
+If you want to use the {dataframe-transforms} feature, you must have the required 
```

Comment:
I suggest that we move this content to the bottom of the page so we don't scare off readers :), similar to how it was done in https://www.elastic.co/guide/en/kibana/master/tutorial-sample-data.html